otter/integration/tests/test_convergence.py

```diff
@@ -67,6 +67,25 @@ def not_mimic():
     return not bool(os.environ.get("AS_USING_MIMIC", False))
 
 
+def mimic():
+    return not not_mimic()
+
+
+def get_resource_mapping():
+    """
+    Get resource mapping based on the environment settings
+    """
+    res = {'nova': (nova_key,), 'loadbalancers': (clb_key,)}
+    if otter_local_url is not None:
+        res['otter'] = ("badkey", otter_local_url)
```

Comment:
Non-blocking, but recommended: Can you please explain 'badkey' and why it's needed?  Thanks!